.github/workflows/pull_request_secure.yml

```diff
@@ -320,8 +323,7 @@ jobs:
           yq -i e 'del(.content.sources)' apps/docs/antora-playbook.yml
           yq -i e 'del(.site.keys)' apps/docs/antora-playbook.yml
           yq -i e '.content.sources += [{"url": "../../timefold-solver", "start_path": "docs/src"}]' apps/docs/antora-playbook.yml
-          npm ci
-          npm run build -- --filter @timefoldai/docs
+          pnpm build --filter @timefoldai/docs
```

Comment:
This used to be

    pnpm install --frozen-lockfile
    pnpm build --filter @timefoldai/docs